source/i18n/rules.ttl

```diff
@@ -11,11 +21,47 @@ base <https://id.kb.se/>
   :code "skr-1980" ;
   :inCollection </i18n/rule/x0> .
 
+</i18n/rule/x0/kr76> a :LanguageTransformRules ; # :LanguageTranscriptionRules
+  :label "Transkribering enligt KR76 (Kjellberg)"@sv ;
+  :code "x0-kr76" ;
+  :inCollection </i18n/rule/x0> ;
+  :languageFormSpecification [ a :TransformedLanguageForm ;
+      :inLangScript </i18n/script/Latn> ;
+      :fromLangScript </i18n/script/Cyrl> ] ;
+  :seeAlso <https://metadatabyran.kb.se/download/18.6945cdaa174b74a2c361604/1601918194487/Transkriberingsschema_Kyrilliska%20alfabet_2015.pdf> .
+
+# TODO: form OK?
+</i18n/rule/x0/frhngbzs-1ed> a :LanguageTransformRules ;
+  :broader </i18n/rule/m0/alaloc> ;
+  :inCollection </i18n/rule/x0> ;
+  :languageFormSpecification [ a :TransformedLanguageForm ;
+      :inLangScript </i18n/script/Latn> ;
+      :fromLangScript </i18n/script/Cyrl> ] ;
```

Comment:
```suggestion
      :fromLangScript </i18n/script/Arab> ] ;
```

As far as I can tell there is no code for [persian alphabet](https://en.wikipedia.org/wiki/Persian_alphabet) in [ISO 15924](https://unicode.org/iso15924/iso15924-codes.html). `Arab` which in some sense "broader" seems to be used at least in some places around the web. Also an option to leave out `fromLangScript`?

Reply:
We should check with experts, but I *believe* the "broader sense" is at least informative enough and not formally wrong. In fact, according to Wikipedia, the [Persian alphabet](https://en.wikipedia.org/wiki/Persian_alphabet) is based on the [Arabic alphabet](https://en.wikipedia.org/wiki/Arabic_alphabet), where it is stated:

>  For the Arabic script as it is used by all languages, see [Arabic script](https://en.wikipedia.org/wiki/Arabic_script).

If I get that right, that means that Persian, written in the Persian alpabet, is in the Arabic script. Thus, with your fix, this is correct.